packages/fxa-content-server/app/scripts/views/choose_what_to_sync.js

```diff
@@ -82,18 +82,18 @@ const View = FormView.extend(
     },
 
     submit() {
-      const account = this.getAccount();
-      const declinedSyncEngines = this._getDeclinedEngineIds();
-      const offeredSyncEngines = this._getOfferedEngineIds();
+      return Promise.resolve().then(() => {
```

Comment:
I notice quite a few of these in the diff. Would they be more aesthetic as `async` functions instead? (we can use those on the front-end, right?)

Reply:
If we did https://github.com/mozilla/fxa/issues/535, then I'd say "yeah, let's do it" since most users would be on UAs that support async/await.